onnxscript/function_libs/torch_lib/registration.py

```diff
@@ -63,8 +69,27 @@ def items(self) -> Generator[tuple[str, OverloadedFunction], None, None]:
 default_registry = Registry()
 
 
+def _check_and_normalize_names(name: str | tuple[str, ...]) -> tuple[str, ...]:
+    names: tuple[str, ...]
+
+    if isinstance(name, str):
+        names = (name,)
+    else:
+        names = name
+    if not isinstance(names, tuple):
+        raise TypeError(f"Name must be a string or a tuple of strings, got {name}")
+    for name_ in names:
+        if name_.endswith(".default") or not _QUALIFIED_OPERATOR_NAME_REGEX.fullmatch(name_):
+            raise ValueError(
+                f"Invalid name '{name_}'. Must be in the form 'namespace::name.overload' "
```

Comment:
maybe highlight the `.default` is not accepted here too?

Reply:
It’s on the next line?

Reply:
indeed minor, but the message doesn't say `torch.add.default` is not allowed, only that `torch.add` is the default overload. that might mean that both `torch.add` (implicit form) and `torch.add.default` (explicit form) are acceptable as default overload

Reply:
I rephrased the message. Hopefully better now?